plutus-metatheory/src/Untyped/CEK.lagda.md

```diff
@@ -369,12 +369,12 @@ BUILTIN constrData = λ
   ; _ -> inj₁ userError
   }
 BUILTIN mapData = λ
-  { (app base (V-con (list (pair pdata pdata)) xs)) → do 
```

Comment:
There's a lot of trailing whitespace in the Agda code.  I haven't fixed it all here because it would introduce hundreds of lines of irrelevant changes.